src/farcasterd/runtime.rs

```diff
@@ -715,7 +740,10 @@ impl Runtime {
                     } else {
                         let xs = self
                             .pending_swap_init
-                            .entry(ServiceId::Syncer(Coin::Bitcoin, *network))
+                            .entry((
+                                ServiceId::Syncer(Coin::Bitcoin, *network),
+                                ServiceId::Syncer(Coin::Monero, *network),
+                            ))
```

Comment:
These two if/else cases are super hard to read with all the code duplication going on. Not the problem of this PR, but something I found distracting.